build/release/cloudbuild.yaml

```diff
@@ -26,21 +53,96 @@ steps:
     id: build-make-docker
     args: ["build", "-f", "Dockerfile.build", "-t", "make-docker", "."]
 
+  #
+  # pull the main build image if it exists
+  #
   - name: "make-docker"
     id: pull-build-image
     dir: "build"
     env:
-      - "REGISTRY=us-docker.pkg.dev/${PROJECT_ID}/ci"
+      - "REGISTRY=${_REGISTRY}/ci"
     args: ["pull-build-image"]
     waitFor:
       - build-make-docker
 
+  #
+  # Ensure example images exists
+  #
   - name: "make-docker"
     id: test-examples-on-gar
     dir: "build"
     env:
-      - "REGISTRY=us-docker.pkg.dev/${PROJECT_ID}"
+      - "REGISTRY=${_REGISTRY}"
       - DOCKER_RUN_ARGS=--network=cloudbuild
     args: ["test-examples-on-gar"]
 
-timeout: 3600s
+  #
+  # Build all the images
+  #
+  - name: "make-docker"
+    id: build-images
+    dir: "build"
+    env:
+      - "VERSION=${_VERSION}"
+      - "REGISTRY=${_REGISTRY}/release"
+      - "FULL_BUILD=1"
+    args: ["-j", "1", "build"]
```

Comment:
Could we do this in parrallel? (i.e. `-j=4` (or whatever number is the best)? Maybe make things a bit faster?)

Reply:
Tried increasing jobs from 1 to 12. Build is succeeded with 1 job only. Received error if I increase the jobs -> docker: Error response from daemon: pull access denied for agones-build, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.

Reply:
Fair enough. At some point we may want to work out why it's doing that.